patterns/2-structured/start-as-experiment.md

```diff
@@ -4,94 +4,52 @@ Start as an Experiment
+- Democratic decision making within community
+
+Consider designating the end of the experiment a _pivot_, _change_ or _pause_ point to re-evaluate. Also consider establishing a [Review Committee](review-committee.md) to increase the chances of management buy-in through participation. Depending on company culture, it might be helpful to accompany the experiment with appropriate metrics [First Steps With Metrics](../1-initial/introducing-metrics-in-innersource.md). If the projects in the experiment don't provide a direct impact on the companies revenue, consider introducing [Cross-Team Project Valuation](crossteam-project-valuation.md) to highlight their value contributions.
 
 ## Resulting Context
 
 Managers are able to kick start InnerSource for the following reasons:
 
-- The experimental setup eases the need for managers to scrutinize the
-  InnerSource program numbers in the same way that they would for typical
-  projects.
-- The possibility of failure of the experiment is understood and accepted. The
-  personal risk for the supporting managers is minimized.
-- Even in case of a failure, the setup ensures that the company will learn from
-  the experiment.
-- In case of success, the data gathered during the experiment will allow
-  managers to make a longer lasting commitment to InnerSource.
-
-Participants in the InnerSource experiment are now conscious that they have to
-prove to management that InnerSource yields the promised benefits.
-It will therefore help to focus work on those activities which provide the most
-demonstrable value thus increasing the chances of success.
-
-Finally, starting as an experiment makes it much easier to sidestep regulations
-and forces such as tool and process policies which could decrease the chances
-of success.
+- The experimental setup eases the need for managers to scrutinize the InnerSource program numbers in the same way that they would for typical projects.
+- The possibility of failure of the experiment is understood and accepted. The personal risk for the supporting managers is minimized.
+- Even in case of a failure, the setup ensures that the company will learn from the experiment.
+- In case of success, the data gathered during the experiment will allow managers to make a longer lasting commitment to InnerSource.
+
+Participants in the InnerSource experiment are now conscious that they have to prove to management that InnerSource yields the promised benefits. It will therefore help to focus work on those activities which provide the most demonstrable value thus increasing the chances of success.
+
+Finally, starting as an experiment makes it much easier to sidestep regulations and forces such as tool and process policies which could decrease the chances of success.
 
 ## Related Patterns
 
-- _Trial Run_ (from the book [_Fearless
-  Change_](https://fearlesschangepatterns.com/))
+- _Trial Run_ (from the book [Fearless Change](https://fearlesschangepatterns.com/))
 
 ## Known Instances
```

Comment:
@gruetter I am wondering about the significance of "(globally distributed software development)" here? 

Could you explain anything about this Known Instance in more detail, to allow people to learn more from it? Could e.g. be if at Bosch you have written anything publicly about how the InnerSource initiative started, or similar. Or maybe a recording of a presentation you or others have given at a previous ISC Summit? (not sure if such exist)